app/web/features/communities/EditCommunityInfoPage.tsx

```diff
@@ -176,7 +176,7 @@ export default function EditCommunityPage({
           </>
         ) : (
           <Redirect
-            to={routeToCommunity(community.communityId, community.slug, "info")}
+            to={routeToCommunity(community.communityId, community.slug)}
```

Comment:
I'd leave this how it was. This one seems to be more for security, so that if someone manually goes to `community/edit`. but they're not an Admin, it will redirect them back.